docker-compose.yml

```diff
@@ -203,6 +220,45 @@ services:
     ports: []
     env_file: []
 
+  fregepoc-flower-prod:
+    <<: *backend
+    container_name: fregepoc-flower-prod
+    command: celery -A fregepoc flower
+    ports:
+      - 5555:5555
+    expose:
+      - 5555
+    environment:
+      CELERY_BROKER_URL: redis://fregepoc-redis
+      CELERY_RESULT_BACKEND: redis://fregepoc-redis
+    depends_on:
+      - fregepoc-celery-worker-prod
+      - fregepoc-celery-downloads-worker-prod
+      - fregepoc-celery-crawl-worker-prod
+      - fregepoc-redis
+    profiles: [ "prod" ]
+
+  fregepoc-celery-crawl-worker-prod:
+      <<: *backend
+      container_name: fregepoc-celery-crawl-worker-prod
+      ports: []
+      command: celery -A fregepoc worker -Q crawl -l info --concurrency=1 -n worker_crawl
+      profiles: [ "prod" ]
+
+  fregepoc-celery-downloads-worker-prod:
+    <<: *backend
+    container_name: fregepoc-celery-downloads-worker-prod
+    ports: []
+    command: celery -A fregepoc worker -Q downloads -l info --concurrency=3 -n worker_downloads
+    profiles: [ "prod" ]
+
+  fregepoc-celery-worker-prod:
+    <<: *backend
+    container_name: fregepoc-celery-worker-prod
+    ports: []
+    command: celery -A fregepoc worker -Q celery -l info --concurrency=6 -n worker_celery
+    profiles: [ "prod" ]
```

Comment:
Hard-coding `concurrency` settings will not make for an easy configuration/scaling in different environments; by design the `docker-compose.yml` is not meant to be edited during deployment in our project (this constraint greatly reduces the effort required during configuration).

I suggest redefining these settings (mb. as ratios for different workers? - this could simplify the config to a single env var) as environment variables.

Reply:
Do you mean we should be using env vars to set the concurrency level in the docker-compose itself or somehow configure the workers from the django app itself? 

The latter seems possible (though not with `concurrency` but `autoscaling`) but may be a bit too overkill.

Reply:
That's what i meant (setting the concurrency level in docker-compose via env vars), but considering that different workers utilize different numbers of threads i was thinking about figuring a way of configuring a single var and scaling it by worker ratios (which most likely is an overkill - we can use 3 different vars for that).

The latter might not be an overkill, but will not preserve the ratios that we've set (it might even exceed the number of logical cores, which would be inefficient, due to celery worker containers not communicating).

Reply:
I'd say let's just have these as 3 env vars in docker-compose for now, and perhaps consider more ergonomic approach like the ratio or something else in other PR?